tests/integration/vendor/rembrandt.js

```diff
@@ -0,0 +1,7 @@
;var y=function(){function e(t){r(this,e),this._imageA=null,this._imageB=null,this._options=s.default.defaults(t,{imageA:null,imageB:null,thresholdType:e.THRESHOLD_PERCENT,maxThreshold:.01,maxDelta:.02,renderComposition:!1,compositionMaskColor:g.default.RED,maxOffset:0}),this._validateOptions()}return a(e,[{key:"compare",value:function(){var e=this;return this._loadImages().then(function(){var t=new _.default(e._imageA,e._imageB,e._options);return t.compare()})}},{key:"_validateOptions",value:function(){var t=this,n=function(e){var n=t._options[e];if(!("string"==typeof n||Buffer.isBuffer(n)||n instanceof m.default))throw new Error("Option `"+e+"` must either be a String, Buffer or Rembrandt.Image.")};n("imageA"),n("imageB");var i=this._options,r=i.thresholdType,a=i.threshold,o=[e.THRESHOLD_PERCENT,e.THRESHOLD_PIXELS];if(o.indexOf(r)===-1)throw new Error("`thresholdType` must be either Rembrandt.THRESHOLD_PERCENT or Rembrandt.THRESHOLD_PIXELS");if(r===e.THRESHOLD_PERCENT&&(a<0||a>1))throw new Error("`threshold` must be between 0 and 1")}},{key:"_loadImages",value:function(){var e=this;return this._loadImage(this._options.imageA).then(function(t){return e._imageA=t,e._loadImage(e._options.imageB)}).then(function(t){e._imageB=t})}},{key:"_loadImage",value:function(e){return new f.default(function(t,n){if(e instanceof m.default)return t(e);if(e instanceof Buffer)return t(m.default.fromBuffer(e));var i=s.default.createImage();i.addEventListener("load",function(){t(m.default.fromImage(i))}),i.crossOrigin="Anonymous",i.src=e})}}],[{key:"createImage",value:function(e,t){return new m.default(e,t)}}]),e}();y.Image=m.default,y.Color=g.default,y.version=n(9).version;for(var b in h.default)y[b]=h.default[b];e.exports=y}).call(t,function(){return this}())},function(e,t){"use strict"},function(e,t,n){"use strict";function i(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}Object.defineProperty(t,"__esModule",{value:!0});var r=function(){function e(e,t){for(var n=0;n<t.length;n++){var i=t[n];i.enumerable=i.enumerable||!1,i.configurable=!0,"value"in i&&(i.writable=!0),Object.defineProperty(e,i.key,i)}}return function(t,n,i){return n&&e(t.prototype,n),i&&e(t,i),t}}(),a=function(){function e(){i(this,e)}return r(e,null,[{key:"defaults",value:function(e){var t={};for(var n in e)t[n]=e[n];for(var i=arguments.length,r=Array(i>1?i-1:0),a=1;a<i;a++)r[a-1]=arguments[a];for(var o=0;o<r.length;o++){var u=r[o];for(var s in u)"undefined"==typeof t[s]&&(t[s]=u[s])}return t}},{key:"createCanvas",value:function(e,t){var n=void 0;return n=document.createElement("canvas"),n.width=e,n.height=t,n}},{key:"createImage",value:function(){var e=void 0;return e=new window.Image}}]),e}();t.default=a},function(e,t,n){var i;(function(r){"use strict";var a="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e};/*!
```

Comment:
@charliesantos Can this be an npm package in `devDependencies`?

Reply:
Unfortunately, the npm package for this module is breaking our builds. It's trying to compile a dependency that is not compatible with latest node that is why I grabbed their pre-built file. We can fork the npm package and fix it but I'm not sure if that's worth it since this is only used in the tests.